src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/CodeAnalysisPropertyPage.xaml

```diff
@@ -0,0 +1,88 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information. -->
+<Rule Name="CodeAnalysisPropertyPage"
+      Description="Settings"
+      DisplayName="Code Analysis"
+      PageTemplate="generic"
+      Order="800"
+      xmlns="http://schemas.microsoft.com/build/2009/properties">
+
+  <Rule.Categories>
+    <Category Name="AllAnalyzers"
+              DisplayName="All analyzers" />
+
+    <Category Name="NetAnalyzers"
+              DisplayName=".NET analyzers" />
+  </Rule.Categories>
+
+  <Rule.DataSource>
+    <DataSource Persistence="ProjectFile"
+                SourceOfDefaultValue="AfterContext"
+                HasConfigurationCondition="False" />
+  </Rule.DataSource>
+
+  <BoolProperty Name="RunAnalyzersDuringBuild"
+                DisplayName="Run on build"
+                Description="Always run code analysis on build."
+                HelpUrl="https://docs.microsoft.com/visualstudio/code-quality/disable-code-analysis"
+                Category="AllAnalyzers">
+    <BoolProperty.Metadata>
+      <NameValuePair Name="SearchTerms" Value="analyzer;code analyzer;analysis;code analysis" />
```

Comment:
These search terms don't add anything. We already include the property name, display name, description, possible values and current values when computing search results.

Search terms only really make sense when there's a common alias for some property that does not appear anywhere in the text of the property. For example "pointer" for unsafe code.

The same is true for all other search terms here.